userspace/libsinsp/container.cpp

```diff
@@ -84,8 +84,39 @@ const sinsp_container_info::container_mount_info *sinsp_container_info::mount_by
 
 sinsp_container_manager::sinsp_container_manager(sinsp* inspector) :
 	m_inspector(inspector),
-	m_last_flush_time_ns(0)
+	m_last_flush_time_ns(0),
+	m_unix_socket_path(string(scap_get_host_root()) + "/var/run/docker.sock"),
+	m_api_version("/v1.24")
 {
+#ifndef CYGWING_AGENT
+	curl_global_init(CURL_GLOBAL_DEFAULT);
```

Comment:
Now that both mesos and the container object might need to initialize libcurl, do you want to have a single global that does this outside of any particular object? (Not really necessary, but it would be a nice to have).

Reply:
I agree, will do